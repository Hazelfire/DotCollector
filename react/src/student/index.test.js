import React from 'react';
import renderer from 'react-test-renderer';
import Student from './index';

test('renders the student screen', () => {
    const component = renderer.create(
      <Student />,
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
