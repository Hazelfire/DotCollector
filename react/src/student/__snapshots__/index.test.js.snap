// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the student screen 1`] = `
<div
  className="ui center aligned container trial"
>
  <img
    className="ui centered image"
    src="test-file-stub"
  />
  <div
    className="ui container"
  >
    <div
      className="ui huge header primary-text"
      style={
        Object {
          "fontSize": "4em",
        }
      }
    >
      Dot Collector
    </div>
    <p
      className="primary-text subtitle"
    >
      Please input the session code
    </p>
    
    <form
      action={undefined}
      className="ui form"
      onSubmit={[Function]}
    >
      <div
        className="field"
      >
        <div
          className="ui medium input"
          content={undefined}
          style={
            Object {
              "width": "100px",
            }
          }
        >
          <input
            disabled={undefined}
            name="givenAccessCode"
            onChange={[Function]}
            required={undefined}
            style={
              Object {
                "textAlign": "center",
              }
            }
            tabIndex={undefined}
            type="text"
            value=""
          />
        </div>
      </div>
       
    </form>
  </div>
</div>
`;
